src/components/DisplayArticlesList.jsx

```diff
@@ -6,7 +6,7 @@ import { useDispatch, useSelector } from "react-redux";
 
 const DisplayArticlesList = () => {
   const dispatch = useDispatch();
-  const { newsFeed } = useSelector(state => state);
+  const { newsFeed } = useSelector((state) => state);
```

Comment:
Might be able to clean up code, remove () around ```(state)```.

Reply:
Removed it :)